packages/react/package.json

```diff
@@ -57,7 +57,6 @@
   "dependencies": {
     "@aws-amplify/ui": "4.0.0",
     "@aws-amplify/ui-react-core": "1.0.3",
-    "@aws-amplify/ui-react-v1": "npm:@aws-amplify/ui-react@1.2.9",
```

Comment:
We also want to remove `legacy` from `  "files": [
    "dist",
    "legacy",
    "internal",
    "LICENSE"
  ],` ?